### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary
This pull request removes duplicate table of contents from the readme. Additionally it removes exception that Disco V2 is only for CP4D and updates the discovery link.

### CODE REVIEW

**Review by kevinkowa — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -316,7 +296,7 @@ We'd love to highlight cool open-source projects that use this SDK! If you'd lik
 [dw-answers]: https://developer.ibm.com/answers/questions/ask/?topics=watson
 [stack-overflow]: http://stackoverflow.com/questions/ask?tags=ibm-watson
 
-[discovery]: https://www.ibm.com/watson/developercloud/discovery/api/v1/
+[discovery]: https://www.ibm.com/cloud/watson-discovery
```
Comment (SirSpidey) [inline README.md:299]: This link definition is never used. And the link that is used is wrong at line 38

**Review by SirSpidey — CHANGES_REQUESTED** [review]
Changes here are correct, but don't catch the link at line 38 or so. Goes to different place than others. Probably should either use the link definition at the bottom or get rid of those if they're not used.

**Comment by mediumTaj** [comment]
Updated and waiting for CI - @SirSpidey can you take a look again to make sure i caught everything?

File: README.md [hunk 2]
```diff
@@ -52,14 +35,14 @@ This SDK provides classes and methods to access the following Watson services:
 
 * [Assistant](https://www.ibm.com/cloud/watson-assistant)
 * [Compare Comply](https://www.ibm.com/cloud/compare-and-comply)
```
Comment (SirSpidey) [inline README.md:37]:
```suggestion
* Compare Comply (deprecated)
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
some links not correct. Also deprecated services don't have product marketing pages.

**Comment by SirSpidey** [comment]
@mediumTaj Just noticed that the Swift SDK has a treatment for the deprecated services that you might just duplicate here.

**Review by SirSpidey — APPROVED** [review]
👍

**Pull request merged**